install-dev/data/xml/tab.xml

```diff
@@ -439,6 +439,9 @@
                 <tab id="Multistore" id_parent="Advanced_Parameters" active="0" enabled="1" hide_host_mode="0" icon="">
                     <class_name>AdminShopUrl</class_name>
                 </tab>
+                <tab id="FeatureFlag" id_parent="Advanced_Parameters" active="1" enabled="1" hide_host_mode="0" icon="">
+                  <class_name>AdminFeatureFlag</class_name>
```

Comment:
Don't forget to edit `install-dev/langs/en/data/tab.xml` to add the wording

Reply:
Fixed in https://github.com/PrestaShop/PrestaShop/pull/23703/commits/410c7ff73ba8a82c41dbed94cb5403810fe27d8e